### PR TITLE
For #311 : Puzzle correction

### PR DESCRIPTION
**Pull request by paulodamaso**
For #311 : Puzzle correction

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#469](https://github.com/timelesslounge/timelessis/issues/469) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#469](https://github.com/timelesslounge/timelessis/issues/469) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/a294dc0e-034e-4b09-8a73-179454558574); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by paulodamaso** [comment]
@emilianodellacasa Please takea look now

**Comment by krzyk** [comment]
@paulodamaso didn't you create an ItemCreateView? That matches with the previous puzzle, not with the UpdateView. Or was it more complex and the names don't match up with the todos?

File: timeless/views.py [hunk 1]
```diff
@@ -8,9 +8,13 @@ class UserView(AdminView):
 @todo #311:30min Once CreateView is implemented, refactor all blueprint views
  to use it for validating the form and storing the record in the database.
  Views already implemented: ItemCreateView
-@todo #173:30min Once UpdateView is implemented, refactor all blueprint views
+@todo #311:30min Once UpdateView is implemented, refactor all blueprint views
  to use it for validating the form and updating the record in the database.
  Reuse SingleObjectMixin to provide simple solution to fetch by id.
+ Views already implemented: ItemCreateView
+@todo #173:30min Once DeleteView is implemented, refactor all blueprint views
```
Comment (krzyk) [inline timeless/views.py]: @paulodamaso hmm, this puzzle was already removed, you are trying necromancy? :)

File: timeless/views.py [hunk 2]
```diff
@@ -8,9 +8,13 @@ class UserView(AdminView):
 @todo #311:30min Once CreateView is implemented, refactor all blueprint views
```
Comment (krzyk) [inline timeless/views.py]: @paulodamaso this todo was already removed (in #310), so I'm not sure if we should leave it here or close it in the issue created from this puzzle (https://github.com/timelesslounge/timelessis/issues/443)?

Comment (paulodamaso) [reply timeless/views.py]: @krzyk Let's remove it, it was already resolved. I've opened an issue to @emilianodellacasa stating that we have too much overlapping and duplicated puzzles for views

File: timeless/views.py (hunk 1 above)
Comment (paulodamaso) [reply timeless/views.py]: @krzyk Well looks like that the ritual didn't worked... :) I'm confused about all these puzzles concerning the views, I've already opened an issue

**Comment by paulodamaso** [comment]
@krzyk Corrections done, please take a look

**Comment by krzyk** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by krzyk** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/timelesslounge/timelessis/issues/469#issuecomment-469676570))

@krzyk This is what I know about this job in [CF9LG1YCA](https://www.0crat.com/p/CF9LG1YCA), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#469](https://github.com/timelesslounge/timelessis/issues/469) is [in scope](http://www.zerocracy.com/policy.html#14) for 20hr
  * The role is `REV`
  * The job is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk) for [20hr](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @krzyk[/z](https://www.0crat.com/u/krzyk) will get 25% of their hourly rate for this job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @paulodamaso[/z](https://www.0crat.com/u/paulodamaso): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/CF9LG1YCA?q=%7Bjob%3A%27gh%3Atimelesslounge%2Ftimelessis%23469%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
Job was finished in 16 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#469](https://github.com/timelesslounge/timelessis/issues/469) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by krzyk** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +6 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**